massa-node/base_config/gas_costs/abi_gas_costs.json

```diff
@@ -1,49 +1,50 @@
 {
   "assembly_script_abort": 0,
-  "assembly_script_address_from_public_key": 275,
-  "assembly_script_append_data": 200,
-  "assembly_script_append_data_for": 224,
-  "assembly_script_call": 7752,
-  "assembly_script_caller_has_write_access": 134,
-  "assembly_script_create_sc": 410,
-  "assembly_script_date_now": 70,
-  "assembly_script_delete_data": 163,
-  "assembly_script_delete_data_for": 186,
-  "assembly_script_function_exists": 25302,
-  "assembly_script_generate_event": 142,
-  "assembly_script_get_balance": 125,
-  "assembly_script_get_balance_for": 162,
-  "assembly_script_get_bytecode": 155,
-  "assembly_script_get_bytecode_for": 206,
-  "assembly_script_get_call_coins": 123,
-  "assembly_script_get_call_stack": 264,
-  "assembly_script_get_current_period": 130,
+  "assembly_script_address_from_public_key": 294,
+  "assembly_script_append_data": 198,
+  "assembly_script_append_data_for": 231,
+  "assembly_script_call": 10000,
+  "assembly_script_caller_has_write_access": 146,
+  "assembly_script_create_sc": 0,
```

Comment:
are we sure of this one ?

Reply:
this needs a followup issue

Reply:
Aside from the zeros, the changes seem inconsistant to me. E.g. assembly_script_call going from 7752 to 193111. Maybe the results are too sensitive to the inputs.
I'll open an issue on the gas calibration repo.

Reply:
Issue here: https://github.com/massalabs/gas-calibration/issues/4